### PR TITLE
Add Multi-storage Transaction sample

### PR DESCRIPTION
**Pull request by brfrn169**
This PR adds Multi-storage Transaction sample. Please take a look!

### CODE REVIEW

File: multi-storage-transaction-sample/wait-for-it.sh [hunk 1]
```diff
@@ -0,0 +1,182 @@
...
+
+if [[ $WAITFORIT_CLI != "" ]]; then
+    if [[ $WAITFORIT_RESULT -ne 0 && $WAITFORIT_STRICT -eq 1 ]]; then
+        echoerr "$WAITFORIT_cmdname: strict mode, refusing to execute subprocess"
+        exit $WAITFORIT_RESULT
+    fi
+    exec "${WAITFORIT_CLI[@]}"
+else
+    exit $WAITFORIT_RESULT
+fi
```
Comment (feeblefakie) [inline multi-storage-transaction-sample/wait-for-it.sh:182]: Can you fix the no newline thing?

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!
I found `no newline ...` in one file, so it would be great if you can fix it just in case.

File: multi-storage-transaction-sample/README.md [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+### Initial data
+
+You first need to load initial data with the following command:
+```
+# ./gradlew run --args="LoadInitialData"
+```
+
+And the following data will be loaded:
+
+- For the `cusomer.customers` table:
```
Comment (Torch3333) [inline multi-storage-transaction-sample/README.md:155]:
```suggestion
- For the `customer.customers` table:
```

**Review by Torch3333 — APPROVED** [review]
LGTM, thank you. 
I found a small typo. Please have a look.

**Pull request merged**